src/scss/skin-modern/components/_subtitleoverlay.scss

```diff
@@ -16,6 +16,19 @@
   top: 0;
   transition: bottom $animation-duration-short ease-out;
 
+  * {
```

Comment:
Just to double-check (haven't tested it): This doesn't interfere with our subtitle styling which can be done by a user through the `SettingsPanel`?

Reply:
Yes settings in styling panel works. Just the inline styles that are defined from TTML has higher priority, but I guess that is expected as its inline in closest level and it did behave like that even before this change.

Reply:
Gotch 👍